test/scripts/e2e-kind.sh

```diff
@@ -125,9 +125,6 @@ if [ "$DUALSTACK_CONVERSION" == true ]; then
 fi
 
 if [ "$OVN_GATEWAY_MODE" == "local" ]; then
-  if [ "$SKIPPED_TESTS" != "" ]; then
-    SKIPPED_TESTS+="|"
-  fi
   SKIPPED_TESTS+="should fallback to local terminating endpoints when there are no ready endpoints with externalTrafficPolicy=Local"
```

Comment:
the other tests seem to have a newline that starts them, then groomTestList substitutes the newline for a |

Doesnt this need a newline as well?

Reply:
speaking of which @ricky-rav I should not have to skip this test right? don't we support terminating endpoints feature with ETP=local?

Reply:
@trozet FYI in lgw v6 lane:
```
ginkgo --nodes=10 '--focus=\[Conformance\]|\[sig-network\]' '--skip=\[Feature:Networking-Performance\]|\[Feature:PerformanceDNS\]|Disruptive|DisruptionController|\[sig-apps\]\sCronJob|\[sig-storage\]|\[Feature:Federation\]|should\shave\sipv4\sand\sipv6\sinternal\snode\sip|kube-proxy|should\sset\sTCP\sCLOSE_WAIT\stimeout|named\sport.+\[Feature:NetworkPolicy\]|should\screate\sa\sPod\swith\sSCTP\sHostPort|service.kubernetes.io/headless|should\sresolve\sconnection\sreset\sissue\s#74839|sig-api-machinery|\[Feature:NoSNAT\]|LoadBalancers\sshould|configMap\snameserver|ClusterDns\s\[Feature:Example\]|should\sset\sdefault\svalue\son\snew\sIngressClass|should\sprevent\sIngress\screation\sif\smore\sthan\s1\sIngressClass\smarked\sas\sdefault|validates\sthat\sthere\sis\sno\sconflict\sbetween\spods\swith\ssame\shostPort\sbut\sdifferent\shostIP\sand\sprotocol|\[Feature:Networking-IPv6\]|should\sprovider\sInternet\sconnection\sfor\scontainers\susing\sDNS|\[Feature:Networking-IPv4\]|IPBlock.CIDR\sand\sIPBlock.Except|Network.+should\sresolve\sconnection\sreset\sissue|NetworkPolicy.+should\sallow\segress\saccess\sto\sserver\sin\sCIDR\sblock|\[Feature:.*DualStack.*\]|should\sfallback\sto\slocal\sterminating\sendpoints\swhen\sthere\sare\sno\sready\sendpoints\swith\sexternalTrafficPolicy=Local|\[Serial\]' 
```
so I see the pipe getting added before the should fallback so we are good there I think ^ ?
`[Feature:.*DualStack.*\]|should\sfallback\sto\slocal\sterminating\sendpoints\swhen\sthere\sare\sno\sready\sendpoints\swith\sexternalTrafficPolicy=Local|\[Serial\]' `

Reply:
`speaking of which @ricky-rav I should not have to skip this test anymore right? don't we support terminating endpoints feature with ETP=local?`

Looking at the title of the test, I was going to say that we should wait for this PR of mine https://github.com/ovn-org/ovn-kubernetes/pull/4170 to be merged.

But then I ran the test on KIND both on the PR image and the master image. It works in both cases.  :-o

So I looked at the test case and it's quite bizarre: it tests a terminating endpoint for a service with ETP=local, but the clients from which it connects to the service are all WITHIN the cluster, so ETP=local means nothing since we're not generating any external traffic and that's why my current fix isn't needed for this test after all.
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/service.go#L3043

In the upcoming weeks I should probably write a proper e2e test for this, possibly with Antonio's cloud load balancer for KIND.

Reply:
yeah, let's remove it!

Reply:
done! let's see if it passes on v4 and v6 lanes correctly.